core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-page-tools-seo/store/dot-page-tools-seo.store.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { of } from 'rxjs';
+
+import { DotPageToolsService } from '@dotcms/data-access';
+import { DotPageTool, DotPageToolUrlParams } from '@dotcms/dotcms-models';
+import { getRunnableLink } from '@dotcms/utils';
+import { mockPageTools } from '@dotcms/utils-testing';
+
+import { DotPageToolsSeoStore } from './dot-page-tools-seo.store';
+
+describe('DotPageToolsSeoStore', () => {
+    let store: DotPageToolsSeoStore;
+    let dotPageToolsService: jasmine.SpyObj<DotPageToolsService>;
+    let pageToolUrlParamsTest: DotPageToolUrlParams;
+
+    beforeEach(() => {
+        dotPageToolsService = jasmine.createSpyObj('DotPageToolsService', ['get']);
+        store = new DotPageToolsSeoStore(dotPageToolsService);
+        pageToolUrlParamsTest = {
+            currentUrl: '/blogTest',
+            requestHostName: 'localhost',
+            siteId: '123',
+            languageId: 1
+        };
+    });
+
+    it('should load page tools', () => {
+        const pageTools: DotPageTool[] = mockPageTools.pageTools.map((tool) => {
+            return {
+                ...tool,
+                runnableLink: getRunnableLink(tool.runnableLink, pageToolUrlParamsTest)
+            };
+        });
+        dotPageToolsService.get.and.returnValue(of(pageTools));
+
+        store.getTools(of(pageToolUrlParamsTest));
```

Comment:
You don't need to call this with an Observable because the effect will turn it into observable, you need to call this the same way the component will call it.